src/screens/MessageScreen.tsx

```diff
@@ -41,23 +49,26 @@ export function MessageScreen({
 
   const renderLeft = useCallback(
     (props: { style: ViewStyle }, user: User) => (
-      <View style={props.style}>
+      <View
+        style={
+          user.isUnread ? [props.style, styles.newMessageIconView] : props.style
```

Comment:
I agree. I removed it since the badge wasn't in the latest designs but I think making the badge show by default and allow for hiding it would be preferable. I'll update in the next PR.